tests/core/asyncio/test_subscription_wait_apis.py

```diff
@@ -1,90 +0,0 @@
-import asyncio
-
-import pytest
-
-from helpers import DummyRequestPair, DummyResponse
-
-
-@pytest.mark.asyncio
-async def test_wait_until_any_subscribed(triplet_of_endpoints):
-    endpoint1, endpoint2, endpoint3 = triplet_of_endpoints
-
-    await endpoint1.subscribe(DummyRequestPair, lambda _: None)
-
-    await endpoint3.wait_until_any_remote_subscribed_to(DummyRequestPair)
-
-    subscriptions = endpoint3.get_connected_endpoints_and_subscriptions()
-
-    assert (endpoint1.name, {DummyRequestPair}) in subscriptions
-
-    assert endpoint3.is_remote_subscribed_to(endpoint1.name, DummyRequestPair)
-    assert not endpoint3.is_remote_subscribed_to(endpoint2.name, DummyRequestPair)
-    assert not endpoint3.is_remote_subscribed_to("nonexistent", DummyRequestPair)
-
-    assert not endpoint3.is_remote_subscribed_to(endpoint1.name, DummyResponse)
-    assert not endpoint3.is_remote_subscribed_to(endpoint2.name, DummyResponse)
```

Comment:
The `test_asyncio_wait_until_any_remote_subscribed_to` test would be a more thorough replacement of this test if it included a `assert client.is_any_remote_subscribed_to(WaitSubscription)` at the end. Currently it's testing that the relevant call doesn't hang forever, `return` would pass that test but it wouldn't pass this one.